runtime/compiler/codegen/J9CodeGenerator.hpp

```diff
@@ -298,21 +298,41 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
    TR_BitVector *setLiveMonitors(TR_BitVector *v) {return (_liveMonitors = v);}
 
 public:
+   /*
+    * \brief
+    *    Get the class or the superclass of the monitor object.
```

Comment:
Wouldn't this be better phrased as get the most abstract type the monitor may be operating on?

Reply:
Do you mean the most concrete type the monitor may be operating on?

Reply:
Depends on how you view it - it is the most specific type that could cover all possible types that could reach the monitor right?

Reply:
Yea, I got it. Changed the phrasing as suggested. Thanks!